README.md

```diff
@@ -67,6 +67,22 @@ A sub span created as the children of root span links to its parent with `Contex
 subSpan, newCtx, err := tracer.CreateLocalSpan(ctx)
 ```
 
+## Get correlation
+
+Get custom data from tracing context.
+
+```go
+value := go2sky.GetCorrelation(ctx, key)
+```
+
+## Put correlation
+
+Put custom data to tracing context.
+
+```go
+go2sky.PutCorrelation(ctx, key, value)
```

Comment:
I prefer to add a bool result to show whether putting is successful. Users could leverage it to throw a panic, log an error message or ignore it at all.

Reply:
Thanks, has added.